gulpfile.js

```diff
@@ -4,11 +4,20 @@ const { watch, dest, src, series, parallel } = require('gulp');
 const sass = require('gulp-sass');
 const sourcemaps = require('gulp-sourcemaps');
 const sassGlob = require('gulp-sass-glob');
+const stylelint = require('gulp-stylelint');
 const postcss = require('gulp-postcss');
 const del = require('del');
 const config = require('./patternlab-config.json');
 const patternlab = require('@pattern-lab/core')(config);
 
+function lintStyles() {
+  return src('**/*.scss', { cwd: './source' })
```

Comment:
If you add a `since:` property to this line (see [`lastRun()`](https://gulpjs.com/docs/en/api/lastrun)), you can get incremental linting in watch mode - it'll only lint changed files.